### PR TITLE
Add Windows Installation

### PR DESCRIPTION
**Pull request by JShep1**
A retarget of https://github.com/ignitionrobotics/ign-gui/pull/161 to Citadel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,16 +31,14 @@ which can be used to add custom widgets.
...
 
     * [Building from Source](#markdown-header-building-from-source)
 
 [Usage](#markdown-header-usage)
 
-[Documentation](#markdown-header-documentation)
-
-[Testing](#markdown-header-testing)
-
 [Folder Structure](#markdown-header-folder-structure)
```
Comment (chapulina) [inline README.md:42]: folder structure is repeated

File: README.md [hunk 2]
```diff
@@ -31,16 +31,14 @@ which can be used to add custom widgets.
...
 
 [Code of Conduct](#markdown-header-code-of-conduct)
```
Comment (chapulina) [inline README.md:44]: Should we remove the install instructions from the README?

**Review by chapulina — COMMENTED** [review]
Nice! I just have a couple comments about the README

File: README.md (hunk 1 above)
Comment (JShep1) [reply README.md:32]: Resolved f1610c19f83028910c2a08c48ab26044e71ac3ae

File: README.md (hunk 2 above)
Comment (JShep1) [reply README.md:34]: Yep, resolved here f1610c19f83028910c2a08c48ab26044e71ac3ae

**Review by chapulina — APPROVED** [review]

**Pull request merged**